unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/LoadingScreen/Scripts/LoadingScreenView.cs

```diff
@@ -17,26 +20,41 @@ public override void Start()
         {
             base.Start();
             showHideAnimator.OnWillFinishStart += OnFadeInFinish;
+            showHideAnimator.OnWillFinishHide += OnFadeOutFinish;
+
+            FadeIn(true);
```

Comment:
Also in connection to the previous comment it should be invoked from the controller, thus you will be able to invoke `TipsController` from the right place.

Reply:
Done as well when moving the tips controller management to the loading screen controller